drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/strategies/DMNDataManagementStrategy.java

```diff
@@ -57,9 +59,13 @@ public void populateRightPanel(final RightPanelView.Presenter rightPanelPresente
             getSuccessCallback(rightPanelPresenter, scenarioGridModel).callback(factModelTreeHolder.getFactModelTuple());
         }
         else {
-            dmnTypeService.call(getSuccessCallback(rightPanelPresenter, scenarioGridModel),
-                                getErrorCallback(rightPanelPresenter))
-                    .retrieveType(currentPath, dmnFilePath);
+            try {
+                dmnTypeService.call(getSuccessCallback(rightPanelPresenter, scenarioGridModel),
+                                    getErrorCallback(rightPanelPresenter))
+                        .retrieveFactModelTuple(currentPath, dmnFilePath);
+            } catch (Exception e) {
```

Comment:
This is a remote method call, I don't think the exception is actually fired. I expect it will be wrapped and just invoked `getErrorCallback` in case of error

Reply:
@danielezonca 
That exception is thrown by backend method 
_ FactModelTuple retrieveFactModelTuple(Path path, String dmnPath) throws Exception;_
so it must be catched.

Reply:
@gitgabrio 
Yes but it is not "true". This method is on client side and it call remotely the backend method. I'm quite sure even if you throw that exception, it will be not propagated to the client as exception. I expect it will invoked the error `getErrorCallback(rightPanelPresenter)`.
So at the end it is misleading that `try { .. } catch` because you are not catching it

Reply:
@danielezonca 
_FIXED_